editor-extensions/vscode/client/src/extension.ts

```diff
@@ -110,7 +106,6 @@ const startClient = (workspace_uri: Uri) => {
 
   let run_path = executablePath;
   let localRunArgs = [...runArgs];
-  let localDebugArgs = [...debugArgs];
```

Comment:
I'm not sure about removing the debug version.  The code samples all have it, even if it's identical now is it worth keeping for potential future changes?

Reply:
I read the typescript typings and it seems more correct to be passing a single command. We don't have a different way to run the LSP in debug mode.